test/test_basal.cpp

```diff
@@ -0,0 +1,137 @@
+//#include <int32_ttypes.h>
+
+#include "gtest/gtest.h"
+#include "basal.h"
+#include "helper.h"
+
+using namespace Basal;
+
+TEST(TestBasalLookUp, NullSchedule)
+{
+  Helper helper;
+  time_t now;
+  int32_t result;
+  result = BasalLookup(helper.GetSchedule(), now);
+
+  EXPECT_EQ(result, -1);
+}
+
+/*Here we are testing if the empty valriable start2Min is initialized with the current time*/
```

Comment:
Try to make sure the test makes this clear instead of needing to comment on it.
The test title may help to explain a bit.